mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/ui/Layout.kt

```diff
@@ -27,3 +29,14 @@ fun SemanticsNode.getBoundsInRoot(): DpRect =
 operator fun DpRect.contains(offset: DpOffset): Boolean {
   return offset.x in left..right && offset.y in top..bottom
 }
+
+/**
+ * Returns true iff the [Offset] is contained within this [Rect] bounds.
+ *
+ * @receiver the bounds of the [Rect].
+ * @param offset the offset that we're checking some bounds for.
+ * @return true if the offset is contained.
+ */
+operator fun Rect.contains(offset: Offset): Boolean {
+  return offset.x in left..right && offset.y in top..bottom
```

Comment:
What is the `..`? Never saw this before

Reply:
This is the `rangeTo` operator ([reference](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-to.html)).

Reply:
Was going to ask the same question, thanks for clearing that up!